### DISCUSSION TITLE
New location for User Data

### DISCUSSION THREAD

**Discussion by TobiasPharos** [Acly/krita-ai-diffusion#448] [Q&A]
Thanks for the Version 1.15.0. It announces that the location of Styles moved to the User's Appdata. It seems like the current version is still looking at the old path too. Is it going to stay like this for future versions? I'm asking because I'm using the plugin in a managed setup and would need to integrate the new path if the old path will be removed. If it stays an addition, it's nice to have the option for user settings now.

**Comment by Acly** [comment]
For styles, the idea is that pre-installed styles will be in the install dir, and user styles can be created in the user dir. So it will keep reading both. Maybe at some point there will be a checkbox to hide pre-installed.

settings.json is different, not sure if that's relevant for your setup.
